doc/Uplink.xml

```diff
@@ -316,6 +316,11 @@
           various mechanisms for streaming Video, Audio and Metadata. This specification defines how
```

Comment:
Propose to completely rework this section instead of appending information. Especially normative requirements must not be put into informative sections like overview.

Reply:
I moved the 'shall' requirement out of introduction (5.4.1) section and copied to 5.4.2/5.4.3, lets discuss tomorrow if this is sufficient, otherwise the cleaner way to move the whole 5.4 section all the way down and label it 'Informative', I am open to feedback and will address as the WG agrees on. I will also try to make a branch on